.github/workflows/main.yml

```diff
@@ -53,9 +53,22 @@ jobs:
         run: yarn build-storybook -o ./storybook-build
         working-directory: flint.ui
 
+  docker:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build the Docker image 🔨
+        run: docker build --file flint.ui/Dockerfile --tag flint.ui:latest flint.ui/
```

Comment:
Refactor this step to use the Docker Build Push: https://github.com/docker/build-push-action (Don't push on the Push/PRs)

Reply:
Got it!